### PR TITLE
API-23713 OAS Changes

### PR DESCRIPTION
**Pull request by jkelly-va**

### CODE REVIEW

File: veteran-verification-api/src/main/java/gov/va/api/lighthouse/veteranverification/api/VeteranVerificationServiceV2.java [hunk 1]
```diff
@@ -871,13 +815,69 @@ public interface VeteranVerificationServiceV2 {
...
+              required = true,
+              schema = @Schema(example = "17020 Tortoise St", description = "Street Address Line"))
+          @QueryParam("street_address_line1")
+          String streetAddressLine1,
+      @Parameter(schema = @Schema(example = " ", description = "Street Address Line 2"))
+          @QueryParam("street_address_line2")
+          String streetAddressLine2,
+      @Parameter(schema = @Schema(example = " ", description = "Street Address Line 3"))
+          @QueryParam("street_address_line3")
+          String streetAddressLine3);
```
Comment (dlennon-va) [inline veteran-verification-api/src/main/java/gov/va/api/lighthouse/veteranverification/api/VeteranVerificationServiceV2.java:874]: Same here, these aren't query parameters, so the docs would come through as incorrect

File: veteran-verification-api/src/main/java/gov/va/api/lighthouse/veteranverification/api/VeteranVerificationServiceV2.java [hunk 2]
```diff
@@ -1432,7 +1495,63 @@ information to find a match (e.g. 'Ann Marie' or 'AnnMarie',
...
```
Comment (dlennon-va) [inline veteran-verification-api/src/main/java/gov/va/api/lighthouse/veteranverification/api/VeteranVerificationServiceV2.java:1554]: Same here

File: veteran-verification-api/src/main/java/gov/va/api/lighthouse/veteranverification/api/VeteranVerificationServiceV2.java [hunk 3]
```diff
@@ -1548,9 +1692,65 @@ information to find a match (e.g. 'Ann Marie' or 'AnnMarie',
...
```
Comment (dlennon-va) [inline veteran-verification-api/src/main/java/gov/va/api/lighthouse/veteranverification/api/VeteranVerificationServiceV2.java:1753]: Same here

File: veteran-verification-api/src/main/java/gov/va/api/lighthouse/veteranverification/api/v2/VeteranAttributes.java
Comment (dlennon-va) [inline veteran-verification-api/src/main/java/gov/va/api/lighthouse/veteranverification/api/v2/VeteranAttributes.java:1]: These shouldnt be marked as query paramters.

**Review by dlennon-va — APPROVED** [review]

**Review by dlennon-va — CHANGES_REQUESTED** [review]
Query parameters

File: veteran-verification-api/src/main/java/gov/va/api/lighthouse/veteranverification/api/v2/VeteranAttributes.java
Comment (jkelly-va) [reply veteran-verification-api/src/main/java/gov/va/api/lighthouse/veteranverification/api/v2/VeteranAttributes.java:1]: Thank you for feedback! Removed them and added to Endpoint Description instead

**Review by dlennon-va — APPROVED** [review]

**Pull request merged**